services/provider/server/consumer.go

```diff
@@ -207,7 +208,7 @@ func (c *ocsConsumerManager) Get(ctx context.Context, id string) (*ocsv1alpha1.S
 	return consumerObj, nil
 }
 
-func (c *ocsConsumerManager) UpdateConsumerStatus(ctx context.Context, id string, status ifaces.StorageClientStatus) error {
+func (c *ocsConsumerManager) UpdateConsumerStatus(ctx context.Context, id string, status *pb.ReportStatusRequest) error {
```

Comment:
why is this being changed? the whole point was to abstract ReportStatusRequest as it keeps on increasing and keep on satisfying StorageClientStatus interface.